component/azstorage/block_blob.go

```diff
@@ -1629,3 +1631,67 @@ func (bb *BlockBlob) SetFilter(filter string) error {
 	bb.Config.filter = &blobfilter.BlobFilter{}
 	return bb.Config.filter.Configure(filter)
 }
+
+func (bb *BlockBlob) updateMetadata(name string, metadata map[string]*string) error {
```

Comment:
changeowner method anyways gets the uid and gid so there is no need to implement another method as this could have been done in that method itself,